test/validate.spec.js

```diff
@@ -133,10 +133,18 @@ describe('screener-runner/src/validate', function() {
         });
     });
 
-    it('should throw error when no apiKey', function(done) {
-      Validate.runnerConfig({})
+    it('should throw error when no apiKey or username', function(done) {
```

Comment:
Can you add a test for the xor? That both apiKey and username/accessKey can't exist at the same time

Reply:
Done